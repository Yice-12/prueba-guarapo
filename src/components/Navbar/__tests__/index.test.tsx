import { render } from "@testing-library/react";
import { Navbar } from "../";

test("render Navbar", () => {
  const { asFragment } = render(<Navbar />);

  expect(asFragment()).toMatchSnapshot();
});
