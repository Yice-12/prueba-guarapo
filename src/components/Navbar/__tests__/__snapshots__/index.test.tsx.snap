// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Navbar 1`] = `
<DocumentFragment>
  <div
    class="nabvar"
  >
    <div
      class="nabvar__content"
    >
      <img
        alt="logoBlack"
        src="[object Object]"
      />
      <ul>
        <li>
          Characters
        </li>
        <li>
          Locations
        </li>
        <li>
          Episodes
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
